### DISCUSSION TITLE
Frigate page reloads when going from portrait to landcape or vicevera. Using Home Assistant on iPhone

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#8966] [Ask A Question]
About 2 weeks ago my iPhone started reloading the Frigate home page whenever I flip from portrait to landscape or the other way around. Page also reloads if I pinch to zoom in on the image.
Say I am watching a clip in portrait mode, I turn the phone 90 degrees and Frigate reverts to Cameras view. This does not happen with any other web page or other apps.
It's happening on my two version of Frigate (12 and 13 beta 6)
Not sure if it's and iPhone or Home Assistant setting.
Any one else experiencing this? 
I've searched but cannot find a solution.

**Comment by NickM-27** [comment]
This sounds like a home assistant issue, 0.12 is many months old and I don't see this behavior at all

**Reply by 18923** [reply]
I'm still trying to figure this out. Just installed the Home Assistant app on my wife's iPhone an the same happens. On an old Android tablet all works as expected.

**Reply by 18923** [reply]
FYI screen capture when rotating iPhone 

https://github.com/blakeblackshear/frigate/assets/59022856/80d70e96-ee26-4b93-9e80-507a981da752

**Reply by 18923** [reply]
Is anyone else experiencing this. I am at a loss to find a solution. It happens on both my 2 iPhones in the house but not on the Android tablet we have.
It only happesns on the Frigate add on for Homeassistant. All other apps (including other addons of Homeassistant) on the iPhone rotate as expected.
Running Homeassistant app on both iPhones (latest firmware) and Android tablet.

**Reply by marky421** [reply]
You are not alone. It's happening for me too and has been for as long as I've been using Frigate on homeassistant. Today is the first time I've been annoyed enough to look for answers. Unfortunately I have none :(

**Reply by 18923** [reply]
FYI 
_"Frigate page reloads when going from portrait to landcape or vicevera"_
Still happens on Version 14 beta 2. 
On my iPhone (iOS v 17.5.1) this only occurs with Frigate web pages.

**Reply by peruihkxt** [reply]
I'm seeing the same behavior on the final release of 0.14 using an Android phone
